File: DKITProject/DAL/Models/Specialty.cs
namespace DKITProject.DAL.Models
{
    public class Specialty
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Announce { get; set; }
        public string Content { get; set; }
        public string ImgIcon { get; set; }
        public string ImgPreview { get; set; }
        public int ControlNumberId { get; set; }
        public ControlNumber ControlNumber { get; set; }
    }
}

File: DKITProject/ClientApp/src/app/models/specialty.ts
export class SpecialtyModel {
    constructor(
        public id: number,
        public name: string,
        public content: string,
        public imgIcon: string
    ){}
}

export class SpecialtyPreviewModel {
    constructor(
        public id: number,
        public name: string,
        public announce: string, 
        public imgIcon: string
    ){}
}

export class AdministrationSpecialityModel {
    constructor(
        public id: number,
        public name: string,
        public announce: string,
        public content: string,
        public imgIcon: string,
        public controlNumberId: number,
        public controlNumber: number,
    ){}
}